### ISSUE TITLE
Nuve createToken API no response to nuveClient

### ISSUE THREAD

**Issue by mntan** [lynckia/licode#1494]
**Description**
Hi all, 

The problem is Nuve createToken API no response to nuveClient. From the licode-demo (https://github.com/lynckia/licode-demos),  I want to have to a token to provide to browser client side but I have got error 
```
503 ErrorError: socket hang up
at createHangUpError (_http_client.js:254:15)
at Socket.socketOnEnd (_http_client.js:346:23)
```

**Steps to reproduce the issue:**
1. Install licode follows to the guide `http://lynckia.com/licode/install.html`
2. Restore a `nuvedb` to mongodb with `16582 rooms element` in a service record for purposing to switch the serviceId, serviceKey in `licode_config.js` from newly generated id, key
3. Test see own video from Chrome Version 78.0.3904.70, and I haven't seen my own video.
and from console.log of posting token request, I got 503 status like as above.

**Describe the results you received:**
I have got this when I called createToken from licode-demos to nuveAPI
```
503 ErrorError: socket hang up
at createHangUpError (_http_client.js:254:15)
at Socket.socketOnEnd (_http_client.js:346:23)
at emitNone (events.js:91:20)
at Socket.emit (events.js:185:7)
at endReadableNT (_stream_readable.js:974:12)
at _combinedTickCallback (internal/process/next_tick.js:74:11)
at process._tickCallback (internal/process/next_tick.js:98:9)
nonon 503 ErrorError: socket hang up
at createHangUpError (_http_client.js:254:15)
at Socket.socketOnEnd (_http_client.js:346:23)
at emitNone (events.js:91:20)
at Socket.emit (events.js:185:7)
at endReadableNT (_stream_readable.js:974:12)
at _combinedTickCallback (internal/process/next_tick.js:74:11)
at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Describe the results you expected:**
I really want that the switch from a (serviceId, serviceKey) to other (serviceId, serviceKey) is successful. So could you guys can help me with my current stuck? how can we fix or any guide for resolving the issue above?

**Licode commit/release where the issue is happening**
I've set up the licode from the commit `cc1f77d384fed2eed91dde2aaf3b86531cf0b7d0`

**Comment by ashar02** [comment]
After installation go to licode folder. Type npm uninstall mongojs. And then npm install mongjs@2.6.0. It will fix your problem. You may change in `nuve/installNuve.sh` as well.

**Comment by lodoyun** [comment]
Thanks for reporting this. @ashar02  is right (thanks). Nuve is not compatible with mongojs 3.0. For now we're forcing 2.6 -> Fix in #1496

**Comment by mntan** [comment]
Thanks @ashar02 so much! I made exactly what you said, then the problem was fixed.
@lodoyun you're awesome, I've seen your fix at #1496, thank you again.
